pkg/native/serializers/serializer_cdx.go

```diff
@@ -68,6 +69,21 @@ func (s *SerializerCDX) Serialize(bom *sbom.Document, _ *native.SerializeOptions
 		return nil, err
 	}
 
+	for _, dt := range bom.Metadata.DocumentTypes {
+		var lfc cdx.Lifecycle
+
+		if dt.Type == nil {
+			lfc.Name = *dt.Name
+			lfc.Description = *dt.Description
+		} else if *dt.Type == sbom.DocumentType_OTHER {
+			lfc.Phase = cdx.LifecyclePhase(strings.ToLower(*dt.Name))
+		} else if *dt.Type != sbom.DocumentType_OTHER {
+			lfc.Phase = cdx.LifecyclePhase(strings.ToLower(dt.Type.String()))
```

Comment:
@puerco that's where I handle the different cases: mapped phase, unmapped phase (OTHER), and custom object.